src/mail/view/MailViewer.ts

```diff
@@ -241,7 +241,19 @@ export class MailViewer implements Component<MailViewerAttrs> {
 				}
 
 				this.rescale(false)
-				if (this.currentlyRenderedMailBody !== sanitizedMailBody) this.renderShadowMailBody(sanitizedMailBody)
+
+				if (this.currentlyRenderedMailBody !== sanitizedMailBody) this.renderShadowMailBody(sanitizedMailBody, attrs)
+				// When conversation information comes in after the primary mail body (which is often),
+				// check if block quote needs to be displayed for the first item in the conversation (position 0 should be a subject)
+				// If this is the first render where we have the conversation position then update all the quotes. We do it to avoid updating the
+				// dom every time.
+				if (this.conversationPositionBodyWasRenderedWith == null && attrs.conversationPosition === 1) {
```

Comment:
is it possible that the item is not at position 1? are deleted items not in the list?

Reply:
Deleted items are not in the list, do you think we should change it to something like `firstDisplayedMail`?

Reply:
that would make it clearer, maybe `firstDisplayableMail` would convey even better that it's not a choice. /closes bikeshed

Reply:
I took it one step further, it was a good suggestion actually